README.md

```diff
@@ -1157,8 +1157,9 @@ For this target to work correctly, either `dart` must be installed on the system
 
 | Option        | Description                                                                                                                                                                                     |
 | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `dartCliPath` | **optional** Path to the Dart CLI. It must be executable by the calling process. Defaults to `dart`.                                                                                            |
-| `packages`    | **optional** List of directories to be released, relative to the root. Useful when a single repository contains multiple packages. When skipped, root directory is assumed as the only package. |
+| `dartCliPath`    | **optional** Path to the Dart CLI. It must be executable by the calling process. Defaults to `dart`.                                                                                            |
+| `packages`       | **optional** List of directories to be released, relative to the root. Useful when a single repository contains multiple packages. When skipped, root directory is assumed as the only package. |
+| `skipValidation` | **optional** Whether to skip validation, which is useful if there are known issues but you need to make a release anyway. |
```

Comment:
this doesn't tell the user what the consequences of this are -- what is validation? what is it validating?

Reply:
```suggestion
| `skipValidation` | **optional** Publish without validation and resolution (this will ignore errors). |
```

copy-pasted what the tool docs say

Reply:
> Update the documentation to reflect real use case and severity of the flag. I.e., *optional* doesn't make it clear when, how, why, and why not to use skipValidation and it feels like a pretty big hammer without guidance.

Does this look good to you or would you like to add something or be more specific about the particalular scenario it's going to be used for this time, @dbennett-sentry ?

```suggestion
| `skipValidation` | **optional** Publishes the package without going through validation steps, such as analyzer & dependency checks. This is useful in particular situations when package maintainers know why the validation fails and wish to side step the issue. For example, there may be analyzer issues due to not following the current (latest) dart SDK recommendation because the package needs to maintain the package compatibility with an old SDK version. 
This option should be used with caution and only after testing and verifying the reported issue shouldn't affect the package. It is advisable to do an alpha pre-release to further reduce the chance of a potential negative impact. |
```